src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
diff normally is ca. 100... 
so if (0 < 100) = true ...
if (-1 < 100) = true ... :p
So Schaka is right.
In conclusion: I suggest to readd the heroic check